src/Goerli-DssSpell.sol

```diff
@@ -59,8 +69,84 @@ contract DssSpellAction is DssAction {
     }
 
     function actions() public override {
+        // --- 2021-12-03 Weekly Executive ---
 
-    
+        // ----------------------------- Rates updates -----------------------------
+        // Increase the ETH-A Stability Fee from 2.5% to 2.75%
+        DssExecLib.setIlkStabilityFee("ETH-A", TWO_SEVEN_FIVE_PCT_RATE, true);
+
+        // Increase the ETH-B Stability Fee from 6.0% to 6.5%
+        DssExecLib.setIlkStabilityFee("ETH-B", SIX_FIVE_PCT_RATE, true);
+
+        // Increase the LINK-A Stability Fee from 1.5% to 2.5%
+        DssExecLib.setIlkStabilityFee("LINK-A", TWO_FIVE_PCT_RATE, true);
+
+        // Increase the MANA-A Stability Fee from 3.0% to 6.0%
+        DssExecLib.setIlkStabilityFee("MANA-A", SIX_PCT_RATE, true);
+
+        // Increase the UNI-A Stability Fee from 1.0% to 3.0%
+        DssExecLib.setIlkStabilityFee("UNI-A", THREE_PCT_RATE, true);
+
+        // Increase the GUSD-A Stability Fee from 0.0% to 1.0%
+        DssExecLib.setIlkStabilityFee("GUSD-A", ONE_PCT_RATE, true);
+
+        // Increase the UNIV2DAIETH-A Stability Fee from 1.5% to 2.0%
+        DssExecLib.setIlkStabilityFee("UNIV2DAIETH-A", TWO_PCT_RATE, true);
+
+        // Increase the UNIV2WBTCETH-A Stability Fee from 2.5% to 3.0%
+        DssExecLib.setIlkStabilityFee("UNIV2WBTCETH-A", THREE_PCT_RATE, true);
+
+        // Increase the UNIV2USDCETH-A Stability Fee from 2.0% to 2.5%
+        DssExecLib.setIlkStabilityFee("UNIV2USDCETH-A", TWO_FIVE_PCT_RATE, true);
+
+        // Increase the UNIV2UNIETH-A Stability Fee from 2.0% to 4.0%
+        DssExecLib.setIlkStabilityFee("UNIV2UNIETH-A", FOUR_PCT_RATE, true);
+
+        // Decrease the GUNIV3DAIUSDC1-A Stability Fee from 0.5% to 0.1%
+        DssExecLib.setIlkStabilityFee("GUNIV3DAIUSDC1-A", ZERO_ONE_PCT_RATE, true);
+
+        // ----------------------------- Debt Ceiling updates -----------------------------
+        // Increase the WBTC-A Maximum Debt Ceiling (line) from 1.5 billion DAI to 2 billion DAI
+        // Increase the WBTC-A Target Available Debt (gap) from 60 million DAI to 80 million DAI
+        DssExecLib.setIlkAutoLineParameters("WBTC-A", 2 * BILLION, 80 * MILLION, 6 hours);
+
+        // Increase the Dust Parameter from 30,000 DAI to 40,000 DAI for the ETH-B
+        DssExecLib.setIlkMinVaultAmount("ETH-B", 40_000);
+
+        // Increase the Dust Parameter from 10,000 DAI to 15,000 DAI for all vault-types excluding ETH-B and ETH-C
+        DssExecLib.setIlkMinVaultAmount("ETH-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("USDC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("USDC-B", 15_000);
+        DssExecLib.setIlkMinVaultAmount("TUSD-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-A", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-B", 15_000);
+        DssExecLib.setIlkMinVaultAmount("WBTC-C", 15_000);
+        DssExecLib.setIlkMinVaultAmount("KNC-A", 15_000);
```

Comment:
Wider discussion needed on whether we need to be changing dust on offboarded collaterals.

Reply:
Derek mentioned these ones:

- BAL-A
- COMP-A
- AAVE-A
- LRC-A
- BAT-A
- ZRX-A
- UNIV2LINKETH-A

Is there anything else?

Reply:
If you can't open new vaults (i.e. DC = 0), I would say we should not update the dust limit.  Increasing the dust limit on an urn has implications about what the owner can do with the urn (i.e. I believe you can't do partial repayment).

Reply:
Is there any ilk in this spell that needs to be removed then?